src/scss/_typography.scss

```diff
@@ -0,0 +1,8 @@
+h2 {
+  margin-bottom: 1.8rem;
```

Comment:
I will remove the margin bottom from the h3 and we can use the utility classes for h3s.